### DISCUSSION TITLE
[Showcase] Add ByteInTheSky to Doks Showcase

### DISCUSSION THREAD

**Discussion by haryps** [gethyas/hyas#471] [Showcase]
### Site name

ByteInTheSky

### Public URL

https://www.byteinthesky.com/

### Short description

It's a programming website focus on .NET technology stack. Currently it is focused on PowerShell.

### Full description

It's a programming website focus on .NET technology stack. Currently it is focused on PowerShell.

### Terms

- [X] I agree to the [Terms of Service](https://gethyas.com/terms/).

**Comment by h-enk** [comment]
Thanks for sharing! When you open Dev Tools, you'll notice some errors in the Console tab that you should fix
